### DISCUSSION TITLE
v2.10.0

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#1458] [Announcements]
## What's Changed

- Add `<leader>lI` mapping to view Null-ls info
- Add `]t` and `[t` mappings to navigate between tabs
- Add `\` and `|` mappings to make horizontal and vertical splits respectively
- Add `heirline.attributes` and `heirline.icon_highlights` tables to allow easily customizing Heirline elements in statusline/winbar/bufferline
- Add new Heirline based bufferline using the new `astronvim.status` API that is used in the `statusline` and `winbar`
  - This feature is opt-in and can be enabled in the `options.g` table with `heirline_bufferline = true` (must run `:PackerSync` afterwards to remove bufferline.nvim
  - [Documentation on enabling Heirline based bufferline](https://astronvim.github.io/Recipes/status#heirline-based-bufferline)
- Don't disable `swapfile` by default (Can be disabled in the `options.opt` table with `swapfile = false`)
- Improved default theme highlights for DAP UI

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v2.9.1...v2.10.0

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v2.10.0'>v2.10.0</a>.</em>

**Comment by mehalter** [comment]
@PumpedSardines any particular reason for the thumbs down? Curious to know if any of the community disagrees with the changes! Can be good insight into the future/direction of the development

**Reply by PumpedSardines** [reply]
I didn't like the \ | mappings :)

**Reply by mehalter** [reply]
Thanks for the feedback! :) It's definitely tough to get mappings that everyone will like all the time, biggest driving force for having a very powerful user configuration setup to change and defaults that we decide.
